scripts/backfill_downloads.py

```
@@ -110,66 +124,111 @@ def get_total_downloads(full_name):
+        with idx_lock:
+            i = worker_idx_counter["i"]
+            worker_idx_counter["i"] += 1
+        _init_worker(i)
+
+    with ThreadPoolExecutor(max_workers=worker_count, initializer=init) as ex:
+        futures = [ex.submit(_process_repo, r, len(repos)) for r in repos]
+        for f in as_completed(futures):
+            # Surface any exception so it doesn't silently swallow failures.
+            try:
+                f.result()
+            except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)